app/controllers/support_interface/sessions_controller.rb

```diff
@@ -4,12 +4,46 @@ class SessionsController < SupportInterfaceController
 
     def new
       session['post_dfe_sign_in_path'] ||= support_interface_path
+      if FeatureFlag.active?('dfe_sign_in_fallback')
+        render :authentication_fallback
+      end
     end
 
     def destroy
       DfESignInUser.end_session!(session)
 
       redirect_to action: :new
     end
+
+    def sign_in_by_email
+      render_404 and return unless FeatureFlag.active?('dfe_sign_in_fallback')
+
+      support_user = SupportUser.find_by(email_address: params.dig(:support_user, :email_address).downcase.strip)
+
+      if support_user
```

Comment:
Does this need the check on UID being present?

Reply:
No! I had it in initially, with a test to check that we correctly dodge if it isn't present, but the test failed because of this https://github.com/DFE-Digital/apply-for-teacher-training/blob/support-magic-link/app/models/support_user.rb#L4

So I removed both the spec + check